graphql.go

```diff
@@ -99,6 +99,14 @@ func UseFieldResolvers() SchemaOpt {
 	}
 }
 
+// Specifies the maximum depth allowed in the query selection. The default is set to 5.
+// This prevents infinite selection of recursive fields
+func MaxDepth(n int) SchemaOpt {
```

Comment:
> comment on exported function MaxDepth should be of the form "MaxDepth ..." (golint)

Reply:
Is that required for Go docs?

Reply:
Not sure but I suspect so. That's the golint error as reported by gometalinter.

Reply:
It is not hard requirement for Go docs, but is for convention.

The first line above an exported identifier will be extracted for brevity in doc tools. For better readability, it is recommended to be __"a complete sentence that begins with the name of the element it describes"__.

To confirm, I tried rendering the Go doc. The first line appears no matter if it starts with the identifier name.

Reply:
To me, it seems like another annoying opinionated thing by Go tooling. More words to type and grow code size.

Reply:
Right, I don't like that one either.